src/commands/synthetics/utils/public.ts

```diff
@@ -885,6 +885,9 @@ export const reportCiError = (error: CiError, reporter: MainReporter) => {
     case 'POLL_RESULTS_FAILED':
       reporter.error(`\n${chalk.bgRed.bold(' ERROR: unable to poll test results ')}\n${error.message}\n\n`)
       break
+    case 'SAFE_DEADLINE_REACHED':
+      reporter.error(`\n${chalk.bgRed.bold(' ERROR: safe deadline reached ')}\n${error.message}\n\n`)
```

Comment:
I'm not sure "safe deadline" is very explicit outside of the code base, could we rephrase that to something more explicit, like "the batch didn't timeout after the expected timeout period"?

Reply:
I think it'd help to have some reference to the batch, and the fact that the batch timeout is unexpectedly running away. What do you think about `UNEXPECTED_BATCH_RUNAWAY`?
But naming is hard and in the code base, we can always add comments to explain what are the concepts behind these names, so I don't really have a strong opinion about that.

Reply:
Oh, maybe `MISSED_BATCH_TIMEOUT`

Reply:
Per discussion, we went with `BATCH_TIMEOUT_RUNAWAY`. The concept of running away exists in other systems: https://www.ibm.com/docs/en/psfa/7.2.1?topic=reference-runaway-query-notification